Tasks/cURLUploader/curluploader.ts

```diff
@@ -86,10 +86,10 @@ async function run() {
             // Now we get a list of all files under this root
             var allFiles = tl.find(findPathRoot);
 
+            filesPattern = filesPattern.replace(/\\/g, '/');
```

Comment:
Can linux paths contain '\'? I don't know. If so, we still need to include the "isWin" check.